common/config/rush/pnpm-lock.yaml

```diff
@@ -295,14 +302,21 @@ importers:
       eslint-plugin-react: ^7.25.1
       eslint-plugin-react-hooks: ^4.2.0
       immer: ^9.0.7
+      jest: ^27.5.1
+      jest-cli: ^27.5.1
+      jest-transform-stub: ^2.0.0
+      msw: ^0.39.2
       react: ^17.0.2
       react-dom: ^17.0.2
       react-redux: ^7.2.0
       react-table: ^7.7.0
       redux: ^4.1.0
       rimraf: ^3.0.2
       simple-react-validator: ^1.6.1
+      ts-jest: ^27.1.3
+      typemoq: ^2.1.0
       typescript: ~4.3.0
+      xmlhttprequest: ^1.8.0
```

Comment:
`xmlhttprequest` is probably no longer used, since you switched to a different localization override.